spec/features/idv/doc_auth/socure_document_capture_spec.rb

```diff
@@ -116,6 +122,22 @@
         end
       end
 
+      context 'invalid request', allow_browser_log: true do
+        context 'getting the capture path w wrong api key' do
+          before do
+            allow(IdentityConfig.store).to receive(:socure_docv_webhook_secret_key).
+              and_return('')
```

Comment:
this above is a valid request
```suggestion
            DocAuth::Mock::DocAuthMockClient.reset!
            stub_docv_document_request(status: 401)
```
we want to flow through handle_invalid_response